__tests__/views/Action.view

```diff
@@ -1,5 +1,6 @@
 Vertical
 onClick props.clickMe
+disabled props.isDisabled ? props.isDisabled : ''
```

Comment:
As for the line of code itself on the test, it would probably read `isDisabled props.isDisabled` or `isDisabled true` (for a fixed value). We can drop the ternary here since `props.isDisabled` is casted as a boolean in any case.